### PR TITLE
Add blog to website

### LINKED ISSUE #730: Add blog to website
I worked on the base of this in `website/src/posts` and `website/src/_includes/layouts/post.njk` but didn't get very far.

The blog will just be a place for us to post announcements, new releases, changelogs etc. Just long form content for the project.

### PR DESCRIPTION
**Pull request by EduardoLopes**
This PR closes #730

- [x] Add page that list all blog posts (auto generated)
  - [x] Sort posts by date
  - [x] Style posts list page
- [x] Show author of the post
- [x] Add page that show all posts from a specific tag
- [x] Show tags on the post page
- [x] Add page that shows all tags
- [x] Style tags
- [x] link the most recent blog post on the top of the homepage
- [x] Add instructions in the website readme

Blog url structure:

`/blog/` Shows all posts
`/blog/posts/{postName}` Shows the blog post
`/blog/tags` Shows all tags
`/blog/tags/{tagName}` Shows all posts tagged with that tag

### CODE REVIEW

File: website/.eleventy.js [hunk 1]
```diff
@@ -57,6 +58,47 @@ module.exports = function(eleventyConfig) {
...
 
+	eleventyConfig.addShortcode(
+		"postslist",
+		function() {
+
+			let files = fs.readdirSync(opts.blogPath);
+
+			//remove index.md from the list and only accept .md files
+			files = files.filter(function(file){
+				return file != 'index.md' && path.extname(file) === '.md';
```
Comment (bitpshr) [inline website/.eleventy.js:69]:
```suggestion
				return file !== 'index.md' && path.extname(file) === '.md';
```

File: website/.eleventy.js [hunk 2]
```diff
@@ -57,6 +58,47 @@ module.exports = function(eleventyConfig) {
...
+			});
+
+			//sort by date
+			files.sort(function(fileA,fileB){
+				const contentA = fs.readFileSync(`${opts.blogPath}/${fileA}`, "utf8").toString();
+				const dateA = contentA.match(/date:(.*)/)[1];
+
+				const contentB = fs.readFileSync(`${opts.blogPath}/${fileB}`, "utf8").toString();
+				const dateB = contentB.match(/date:(.*)/)[1];
+				console.log(new Date(dateB) - new Date(dateA));
```
Comment (bitpshr) [inline website/.eleventy.js:79]:
```suggestion
```

Comment (EduardoLopes) [reply website/.eleventy.js]: all resolved. thanks!

**Comment by EduardoLopes** [comment]
@sebmck i might finish this tomorrow. anything to add or change?

**Comment by sebmck** [comment]
Nope, looks good to me!

**Pull request merged**